grammar/rollem.pegjs

```diff
@@ -173,7 +178,7 @@
   // This is used to configure stylization of the individual die results.
   function dieFormatter(value, size, isKept = true) {
     var formatted = value
-    if (value == size)
+    if (value >= size)
       formatted = maxFormatter(formatted);
     else if (value == 1)
       formatted = minFormatter(formatted);
```

Comment:
This leaves minimum values bolded. Did you intend that?

Reply:
Yep, easily spotting ones is still useful. Especially when 1 auto fails/misses.